jaxrs/src/test/scala/com/cerner/beadledom/jaxrs/exceptionmapping/JsonMappingExceptionMapperSpec.scala

```diff
@@ -51,7 +51,7 @@ class JsonMappingExceptionMapperSpec
         dispatcher.invoke(request, response)
 
         response.getStatus mustBe BAD_REQUEST.getStatusCode
-        response.getOutputHeaders.getFirst(CONTENT_TYPE) mustBe MediaType.APPLICATION_JSON
+        response.getOutputHeaders.getFirst(CONTENT_TYPE) mustBe MediaType.APPLICATION_JSON_TYPE
```

Comment:
Is the change from `APPLICATION_JSON` to `APPLICATION_JSON_TYPE` another change that is going to break downstream consumers of resteasy?

Reply:
Probably, I ll add that to the changelog entry as well.

Reply:
I think technically this new version of Beadledom will still be backward compatible with resteasy 3.6.3.Final since we are only making this change in the tests and the other changes in this PR seem totally passive.

This means downstream consumers could upgrade Beadledom but still manage resteasy to a lower version than 3.11.

Reply:
Yes thats true. c2e3df00 I have added a note in changelog to make it a little more clear.